CHANGES.rst

```diff
@@ -4,7 +4,8 @@ Changelog
 1.3.2 (unreleased)
 ------------------
 
-- Nothing changed yet.
+- Fix collection not filtering integer field (closes `#32`_).
```

Comment:
@rodfersou in fact what you did was add 3 new operators because the current ones did not work as expected; please amend the changelog entry.

and this is a good place to use ``git rebase``, BTW ;-)